### PR TITLE
fix locale for non Iso format definitions like 'de', 'en' in faker.

### PR DESCRIPTION
**Pull request by ckruppe**
fixes an bug for language definitions in faker that don't follow ISO specifications. E.g. 'de', 'en'

### CODE REVIEW

**Comment by jormaechea** [comment]
Hi @ckruppe, thanks for your PR. Good catch, but there will be an issue for the `pt_PT` locale (it’s the only one different in faker)

I would suggest that the util you implemented is explicitly named to work with faker and to make an exception for this locale.

What do you think?

**Comment by ckruppe** [comment]
Good call thank you i will change that. :)

**Review by jormaechea — APPROVED** [review]
Looks good! 🚀

File: lib/utils/get-faker-locale.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const getFakerLocale = () => {
+	const { locale } = Intl.DateTimeFormat().resolvedOptions();
+	const [lang, country] = locale.split('-');
+
+	return (
+		lang.toLowerCase() === country.toLowerCase()
+		&& (lang !== 'pt' && country !== 'PT')
+	) ? lang : `${lang}_${country}`;
```
Comment (jormaechea) [inline lib/utils/get-faker-locale.js:10]:
```suggestion
	const [lang, country = ''] = locale.split('-');

	return (lang.toLowerCase() === country.toLowerCase() && lang.toLowerCase()) === 'pt' ? lang : `${lang}_${country}`;
```

**Pull request merged**